### PR TITLE
move Support Scripts to be LiveViews

### PR DESCRIPTION
**Pull request by joshk**

### CODE REVIEW

File: test/nerves_hub_web/live/support_scripts_test.exs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      |> click_button("Save Changes")
+      |> assert_path("/org/#{org.name}/#{product.name}/scripts/#{script.id}/edit")
+      |> assert_has("span", text: "can't be blank", count: 1)
+      |> fill_in("Script name", with: "MOTDDDDD")
+      |> fill_in("Script text", with: "dbg(NervesMOTD.print())")
+      |> click_button("Save Changes")
+      |> assert_path("/org/#{org.name}/#{product.name}/scripts")
+      |> assert_has("td", text: "MOTDDDDD")
+
+      assert %{text: "dbg(NervesMOTD.print())"} = Scripts.get!(script.id)
```
Comment (jjcarstens) [inline test/nerves_hub_web/live/support_scripts_test.exs:77]: Is this supposed to be dbg?

**Review by jjcarstens — APPROVED** [review]
LGTM - there is some sort of dbg showing in the test that I couldn't find in the code, but other than that looks good

File: test/nerves_hub_web/live/support_scripts_test.exs (hunk 1 above)
Comment (joshk) [reply test/nerves_hub_web/live/support_scripts_test.exs:77]: yes, this is a test on editing the script and you will notice that I add the `dbg(...)` here https://github.com/nerves-hub/nerves_hub_web/pull/1370/files/8c003465e3db0333cab94e891086727091387fac#diff-93e717a7fa34ab953330244e2b2f97ebd3b03d68b4c010bff43e82dea614c4adR72

Comment (jjcarstens) [reply test/nerves_hub_web/live/support_scripts_test.exs:77]: Ah. There it is 🤦

**Pull request merged**